README.md

```diff
@@ -759,7 +847,9 @@ Operands:
 Kind: *Instruction*
 
 Errors:
-- `ExpectationFalse`: In case any of the expectations are broken.
+- `PalletNotFound`
+- `NameMismatch`
+- `VersionIncompatible`
```

Comment:
Unrelated to this changes, but `VersionIncompatible` as an `XcmError` is kind of confusing isn't it? One might think it's thrown when you are speaking different XCM versions.